src/lib/viewers/BaseViewer.js

```diff
@@ -1000,6 +1006,30 @@ class BaseViewer extends EventEmitter {
         return AnnotationMode.NONE;
     }
 
+    /**
+     * Updates localStorage with the state of the ftux cursor toggle
+     *
+     * @protected
+     * @param {string} key
+     * @return {void}
+     */
+    handleFtuxCursorToggle(key) {
+        if (!this.cache.get(key)) {
+            this.cache.set(key, true, true);
```

Comment:
Seems like we might want to pull out marking the ftux status in a separate method